docs/vsphere_ci/answer-files/autounattend.xml

```diff
@@ -113,24 +115,14 @@
                     <Description>Install and configure OpenSSH Server</Description>
                     <CommandLine>cmd.exe /c powershell.exe -File a:\install-openssh.ps1 "a:\authorized_keys"</CommandLine>
                 </SynchronousCommand>
-                <SynchronousCommand wcm:action="add">
-                    <Order>6</Order>
-                    <Description>Configure firewall rules</Description>
-                    <CommandLine>cmd.exe /c powershell.exe -File a:\install-firewall-rules.ps1</CommandLine>
-                </SynchronousCommand>
-                <SynchronousCommand wcm:action="add">
-                    <Order>7</Order>
-                    <Description>Install Windows Patch KB5012637</Description>
-                    <CommandLine>cmd.exe /c powershell.exe -File a:\install-kb5012637.ps1</CommandLine>
-                </SynchronousCommand>
             </FirstLogonCommands>
             <OOBE>
                 <HideEULAPage>true</HideEULAPage>
                 <HideLocalAccountScreen>true</HideLocalAccountScreen>
                 <HideOEMRegistrationScreen>true</HideOEMRegistrationScreen>
                 <HideOnlineAccountScreens>true</HideOnlineAccountScreens>
                 <HideWirelessSetupInOOBE>true</HideWirelessSetupInOOBE>
-                <NetworkLocation>Home</NetworkLocation>
+                <NetworkLocation>Work</NetworkLocation>
```

Comment:
Does this change have any effect in practice?

Reply:
It is a well known albeit deprecated value.